### ISSUE TITLE
False positive with \Symfony\Component\DependencyInjection\ParameterBag\ContainerBagInterface::get()

### ISSUE THREAD

**Issue by alexsakhnov** [phpstan/phpstan-symfony#191]
Symfony `ContainerBagInterface` implements `Psr\Container\ContainerInterface` but actually [it returns a container parameter][1], not a service. 

However `ContainerInterfaceUnknownServiceRule` checks for service existence, not parameter. For example this code

```php
/** @var \Symfony\Component\DependencyInjection\ParameterBag\ContainerBagInterface $params */
$parameterValue = $params->get('parameter_name');
```

results in the following error

```
Service "parameter_name" is not registered in the container.
```

I suggest to ignore subtypes of `ContainerBagInterface` in the `ContainerInterfaceUnknownServiceRule`. Is this correct?

[1]:https://symfony.com/doc/current/configuration.html#accessing-configuration-parameters

**Comment by ondrejmirtes** [comment]
Hi, this is easy to fix. See the instructions: https://phpstan.org/developing-extensions/type-system#querying-a-specific-type

Just adding an `if` with `(new ObjectType(ContainerInterface::class))->isSuperTypeOf($argType)->yes()` should be fine. There are already some exceptions there: https://github.com/phpstan/phpstan-symfony/blob/fe5f5d8db30e1d25a7e874497902bd900d6229ab/src/Rules/Symfony/ContainerInterfaceUnknownServiceRule.php#L62-L69
